packages/special-pages/tests/page-objects/specialerror.js

```diff
@@ -88,8 +90,43 @@ export class SSLErrorPage {
     async visitsSite () {
         const { page } = this
         await page.pause()
-        await page.getByRole('button', { name: 'Advanced...' }).click()
+        this.showsAdvancedInfo()
         await page.getByRole('button', { name: 'Accept Risk and Visit Site' }).click()
         await this.mocks.waitForCallCount({ method: 'visitSite', count: 1 })
     }
+
+    /**
+     * Clicks on help links in Phishing page
+     */
+    async opensPhishingHelpPage () {
+        await this.opensNewPage('Learn more', 'https://duckduckgo.com/duckduckgo-help-pages/')
+        await this.showsAdvancedInfo()
+        await this.opensNewPage('Phishing and Malware Protection help page', 'https://duckduckgo.com/duckduckgo-help-pages/')
+    }
+
+    /**
+     * Clicks on advanced link to show expanded info
+     */
+    async showsAdvancedInfo () {
+        const { page } = this
+        await page.getByRole('button', { name: 'Advanced...' }).click()
+    }
+
+    /**
+     * Clicks on link and expects it to open a URL in a new window
+     *
+     * @param string linkName
+     * @param string newTabURL
+    */
```

Comment:
Can you check if this is valid jsdoc syntax? (try passing something other than a string). I'm not sure if Typescript will pick these up when not wrapping `string` in `{...}`

Reply:
Oops. Fixed!